libraries/EmbeddedChessPieces/src/ECPMovement/ECPMovement.cpp

```diff
@@ -0,0 +1,32 @@
+#include "ECPMovement.h"
+
+ECPMovement::ECPMovement(Dezibot &d, uint mc = 3900) : dezibot(d), ecpColorDetection(d) {
+    movementCalibration = mc;
+};

```

Comment:
```suggestion
ECPMovement::ECPMovement(Dezibot &d, uint mc = 3900)
    : dezibot(d), ecpColorDetection(d), movementCalibration(mc) {};

```

Reply:
This wasn't fixed, yet. Please break the line since it's rather long. We should aim for about 80 characters